splice_machine/src/test/java/com/splicemachine/derby/transactions/TransactionAdminIT.java

```diff
@@ -144,4 +144,43 @@ public void testGetCurrentTransactionWorks() throws Exception{
 //            s.execute("insert into "+table+" (a,b) values (1,1)");
 //        }
     }
+
+    @Test
+    public void testConnection() throws Exception {
+        Properties info = new Properties();
+        String url = "jdbc:splice://localhost:1527/splicedb;user=splice;password=admin;autoCommit=%s";
```

Comment:
To make it a bit more future proof, you could enhance the connection builder in SpliceWatcher to avoid using the direct string here.
Then create the connection with:
```
try (TestConnection conn = methodWatcher.connectionBuilder().autoCommit(true).build()) {
}
```